test/e2e/framework/cfg_manager/config.go

```diff
@@ -0,0 +1,62 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+package cfg_manager
+
+import (
+	"flag"
+	"time"
+)
+
+// YuniKornTestConfigType holds all of the configurable elements of the testsuite
+type YuniKornTestConfigType struct {
+	JsonLogs    bool
+	LogLevel    string
```

Comment:
how these configs are used?
Is it possible to use the logger library we had in `pkg/log` for logging?

Reply:
The idea behind exposing these flags is to provide options for logging in json format and different logging levels. This is not yet feature complete, will push in next patch.

Reply:
yes this will use pkg/log library